### DISCUSSION TITLE
[feature] Double check for sensitive commands.

### DISCUSSION THREAD

**Discussion by StLeoX** [fish-shell/fish-shell#10690] [Ideas]
I want this feature:
- Double check for sensitive commands, like `rm -rf /`
- Supports user adding commands to the block list.

**Comment by faho** [comment]
This has been asked for before, the problem is that the shell cannot tell which commands are dangerous or not.

`rm` *might* be perfectly safe, or it might not be. `apt` might be perfectly safe, or it might not be. There's no good way to make this that isn't full of holes.

Sorry.
